Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM php:8.0-alpine
+FROM php:8.1-alpine
 
-ENV COMPOSER_HOME="/tmp/composer"
+ENV \
+    COMPOSER_ALLOW_SUPERUSER="1" \
```

Comment:
What is this for?

Reply:
removed  `COMPOSER_ALLOW_SUPERUSER="1" \`